assets/src/components/account/GettingStartedOverview.tsx

```diff
@@ -2,32 +2,37 @@ import React from 'react';
 import {Box} from 'theme-ui';
 import * as API from '../../api';
 import Title from 'antd/lib/typography/Title';
-import {Paragraph, Input, colors, Text} from '../common';
+import {Button, Paragraph, Input, colors, Text} from '../common';
 import {TwitterPicker} from 'react-color';
 import SyntaxHighlighter from 'react-syntax-highlighter';
 import ChatWidget from '@papercups-io/chat-widget';
 import {atomOneLight} from 'react-syntax-highlighter/dist/esm/styles/hljs';
 
 type Props = {};
 type State = {
-  account: any;
+  account_id: any;
   color: string;
   title: string;
   subtitle: string;
+  id?: string;
 };
 
 class GettingStartedOverview extends React.Component<Props, State> {
   state: State = {
-    account: null,
+    account_id: null,
```

Comment:
just a nitpick -- i know it's a little annoying to switch back and forth, but in general JS/TS code is camelCase rather than snake_case